### ISSUE TITLE
Wonky behaviour in "Unrecognized" utility

### ISSUE THREAD

**Issue by Baine** [ShokoAnime/ShokoDesktop#551]
## VERSION INFORMATION

Server Version: 
docker cazzar/shokoserver:latest
Desktop Version: 
5c0fa8cafc4567ebd6d7a449285640765902eded

## LOG FILE

no logs since no errors or actions logged by desktop

## DESCRIPTION

Using Utilities -> Unrecognized
After adding a new Group/Series the form used to refresh the series list and filter it by the title of the newly added group.
Currently this does not happen.

Also, the sorting of the series list seems to be a bit strange atm,
see the screenshot below as reference
![image](https://user-images.githubusercontent.com/6566469/30118058-4fa7c2a2-9322-11e7-97fb-b164fd88d72e.png)

## STEPS TO REPRODUCE

Open ShokoDesktop, connect to server of choice
Go to Utilities, select Unrecognized
On the right side, click on new Series, follow the steps in the newly opened form

**Comment by da3dsoul** [comment]
It's not strange, it's sorting by Levenshtein distance to the selected item. The not refreshing is a problem though

**Comment by Baine** [comment]
Something else I've noticed just now.
When manual linking, the last selected group is "lost"

i used to be able to manually link a range of episodes, select a special, switch to single episode, select the special, press ok and done

now i, given that i first selected the group i want the files to link to,
i manually link multiple episodes to a range, have to search for the group and select it again, then can link again

that's especially annoying if multiple files for a group (that aren't in sequential order) have to be added.
example: all same anime, episodes 1,2,3,5,6,7,10, special 01 and special 02
link 1-3, search again
link 5-7, search again
link 10, search again
link special 01, search again
link special 02, search again

I understand if that won't be fixed in the current installment of shoko desktop, but maybe that's something to avoid in any possible v2

(commit: 5c0fa8cafc4567ebd6d7a449285640765902eded)

**Comment by da3dsoul** [comment]
Yeah, I'm not sure how I could avoid that. I can tell you that I am definitely not reverting the background loading. It makes the utilities so much more responsive

**Comment by da3dsoul** [comment]
@baine is this fixed? I know I did plenty of work on it

**Comment by Baine** [comment]
@da3dsoul for the most part, yes

just noticed a case today, that's still a bit weird.
Show: Grimoire of Zero (https://anidb.net/anime/12450)

when searching for it:
![image](https://user-images.githubusercontent.com/6566469/69819177-a5d84180-11fe-11ea-96a0-60c37629c15f.png)

I'd expect it at the very top, but instead it's at the end of the list.
(saw that with other shows too)

that's the only outstanding issue (i wouldn't even really call it an issue) and most likely by how the distance is calculated
